microsoft-edge/webview2/reference/javascript/sharedbufferreceivedevent.yml

```
@@ -0,0 +1,131 @@
+### YamlMime:TSType
+name: SharedBufferReceivedEvent
+uid: 'WebView2Script!SharedBufferReceivedEvent:class'
+package: WebView2Script!
+fullName: SharedBufferReceivedEvent
+summary: >-
+  Event object for the `chrome.webview.sharedbufferreceived` event. This event is dispatched when
+  `CoreWebView2.PostSharedBufferToScript` is successfully called.
+remarks: |-
+
+
+  #### Examples
+
+  For Example, if we want to send data to script for one time read only consumption,

```

```suggestion
The following example sends data to script for one-time, read-only consumption.
```

---

```suggestion
  The following example sends data to script for one-time, read-only consumption.
```
Keeping the indentation.